apps/website/src/content/docs/toast.mdx

```diff
@@ -1,18 +1,100 @@
+  <AllExamples.ToastStatusExample client:load />
+</LiveExample>
+
+For further customization with `options`, you can utilize `link`, which allows taking further actions directly from the toast, such as navigating to another page or triggering a specific function.
+
+<LiveExample src='Toast.link.jsx'>
+  <AllExamples.ToastLinkExample client:load />
+</LiveExample>
+
+### Settings
+
+The object returned by `useToaster` also provides a `setSettings` method, which can be used to customize the `placement` and `order` of toasts.
+
+`placement` supports the following values:
+
+- `"top"` (default)
+- `"top-start"`
+- `"top-end"`
+- `"bottom"`
+- `"bottom-start"`
+- `"bottom-end"`
+
+`"start"` indicates the left side of viewport, while `"end"` represents the right side.
+
+<LiveExample src='Toast.placement.jsx'>
+  <AllExamples.ToastPlacementExample client:load />
+</LiveExample>
+
+`order` supports the following values:
+
+- `"auto"` (default)
+- `"descending"`
+- `"ascending"`
+
+When set to `"descending"`, the most recent toasts are on top. When set to `"ascending"`, the most recent toasts are on bottom. When set to `"auto"`, it will behave like `"descending"` when `placement` is set to a value beginning with `"top"`, otherwise it will behave like `"ascending"`.
+
+<LiveExample src='Toast.order.jsx'>
+  <AllExamples.ToastOrderExample client:load />
+</LiveExample>
+
+### Closing toasts
+
+When setting the `hasCloseButton` option available in the status method to `true`, each toast will be displayed with a close button, which allows you to close the toast manually.
+
+<LiveExample src='Toast.hasCloseButton.jsx'>
+  <AllExamples.ToastHasCloseButtonExample client:load />
+</LiveExample>
+
+Additionally, to implement further actions upon the closing of each toast or to chain one toast after another, you can retrieve the `close()` function returned from the status method.
+
+<LiveExample src='Toast.close.jsx'>
+  <AllExamples.ToastCloseExample client:load />
+</LiveExample>
+
+Closing also depends on the type of toast when used without `hasCloseButton`. By default, persisting toasts will not be closed automatically and will contain a close button while temporary toasts will automatically close after 7 seconds and will not contain a close button. The type of toasts can be specified by passing either `"persisting"` or `"temporary"` into the `type` option.
+
+<LiveExample src='Toast.type.jsx'>
+  <AllExamples.ToastTypeExample client:load />
+</LiveExample>
+
+To close all toasters on viewport at once, use the `closeAll()` method offered by the toaster object.
+
+<LiveExample src='Toast.closeAll.jsx'>
+  <AllExamples.ToastCloseAllExample client:load />
+</LiveExample>
+
+The `animateOutTo` prop allows you to control the direction to which the toast animates out when closing. Specifically, it helps determine an "anchor" element that the toast will go towards when it is dismissed by pointing to a ref passed into that element.
```

Comment:
We can simplify wording and provide an example use-case of this feature.


```suggestion
The `animateOutTo` prop allows you to control the direction to which the toast animates out when closing. Specifically, it helps determine an "anchor" element that the toast will go towards when it is dismissed. This can be useful, for example, when you want to indicate to the user that the toasts are being saved into a "notifications" panel.
```